### PR TITLE
Added modtoberfest

### LINKED ISSUE #1159: [Hacktoberfest] Add Swag Opportunity for Modtoberfest
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : <a href='https://modtoberfest.com/'>Modtoberfest</a>

Q. Where have you seen the announcement (URL)?  
A. : https://modtoberfest.com/modtoberforest

Q. What kind of swag (clothing, stickers, ...)?  
A. : Stickers

Q. Have you received swag from this?  
A. : No

Q. How long did it take?  
A. : N/A

Q. Does it ship worldwide?  
A. : Yes 

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by DevanshKyada27**
Fixes #1159 

Added and updates modtoberfest.

Hope this solves the issue.

Looking forward for merging this PR.

- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Added the modtober fest in the file and updated it.

### CODE REVIEW

**Comment by aslafy-z** [comment]
Please review the CI results

File: data.json [hunk 1]
```diff
@@ -962,4 +962,12 @@
     "dateAdded": "2019-10-08T21:00:00.000Z",
     "tags": ["stickers", "hacktoberfest", "clothing", "expired"]
   }
+   {
+    "name": "Modtoberfest",
+    "difficulty": "medium",
+    "description": " Pull requests that have been scripted or automated. Pull requests that hurt a project more than they help. Pull requests that focus on minor changes such as whitespace, indenting, typos, and optimizing assets and images",
+    "reference": "https://modtoberfest.com/",
+    "dateAdded": "2023-10-15T15:56:00.000Z",
+    "tags": ["stickers", "pins"]
```
Comment (aslafy-z) [inline data.json:971]: Please add the `hacktoberfest` tag

**Comment by BandhiyaHardik** [comment]
@aslafy-z can i work on it ? 

this issue was assigned to me.

**Comment by aslafy-z** [comment]
I've seen @DevanshKyada27 confirm he was ok.  Feel free switch to address some other issue. I will close this PR.

**Pull request closed without merging**